### PR TITLE
add federatedETL.federator.primaryClusterID

### PR DESCRIPTION
**Pull request by jessegoodier**
Add `--set federatedETL.federator.primaryClusterID=$CLUSTER_NAME` per 1.101 update
Add backup considerations

### CODE REVIEW

**Review by bstuder99 — DISMISSED** [review]

File: federated-etl.md [hunk 1]
```diff
@@ -8,7 +8,18 @@ Below is the configuration guide using **Kubecost ETL Federation**
 
 Federated ETL gives teams the benefit of federating multiple Kubecost installations into one view without dependency on Thanos.
 
-For environments that already have a Prometheus instance, this ETL Federation may be preferred because the only dependency will be a single Kubecost pod for monitored cluster.
+There are two primary advantages for using ETL Federation:
+1. For environments that already have a Prometheus instance, Kubecost only requires a single pod per monitored cluster
+2. Many solutions that aggregate Prometheus metrics (like Thanos) often to not scale well in large environments
```
Comment (thomasvn) [inline federated-etl.md:13]: Perhaps adjust this?

> Many solutions that aggregate Prometheus metrics (like Thanos) often to not scale well in large environments

to

> Many solutions that aggregate Prometheus metrics (like Thanos), are often expensive to scale in large environments

**Comment by thomasvn** [comment]
@jessegoodier I'm in the middle of working on a new Federated ETL Backups & Alerting doc. I'll try to have a PR out by end of day.

File: federated-etl.md [hunk 2]
```diff
@@ -8,7 +8,18 @@ Below is the configuration guide using **Kubecost ETL Federation**
 
 Federated ETL gives teams the benefit of federating multiple Kubecost installations into one view without dependency on Thanos.
 
-For environments that already have a Prometheus instance, this ETL Federation may be preferred because the only dependency will be a single Kubecost pod for monitored cluster.
+There are two primary advantages for using ETL Federation:
+1. For environments that already have a Prometheus instance, Kubecost only requires a single pod per monitored cluster
```
[the remark this answers is not in the document]
Comment (jessegoodier) [reply federated-etl.md:12]: As long as the metrics we need are available, this method is more efficient. 

Determining if metrics are missing should be easier. See this: https://github.com/kubecost/cost-analyzer-helm-chart/issues/2066

[the remark this answers is not in the document]
Comment (thomasvn) [reply federated-etl.md:12]: Thanks!

File: federated-etl.md (hunk 1 above)
Comment (jessegoodier) [reply federated-etl.md:13]: much better. will commit. 

Any other changes based on the work you are doing? I'll wait for your approval before committing. 

Not urgent.

**Comment by jessegoodier** [comment]
@thomasvn updated, please review.
I put the link at bottom as this topic is chronologically later in a setup.

File: federated-etl.md [hunk 3]
```diff
@@ -107,3 +112,9 @@ prometheus:
     * To verify the entire pipeline is working, either query `Allocations/Assets` or view the respective views on the frontend. Multi-cluster data should appear after:
         * The Federator has run at least once.
         * There was data in the Federated Storage for the Federator to have combined.
+
+### Data Recovery
+
+When using ETL Federation, there are several methods to recover Kubecost data in the invent of data loss:
+
+The [backup and alerting](https://docs.kubecost.com/install-and-configure/install/multi-cluster/federated-etl/federated-etl-backups-alerting) has detail regarding the various methods.
```
Comment (thomasvn) [inline federated-etl.md:120]: Can we adjust the link here to instead show

```
[backups and alerting](/federated-etl-backups-alerting.md)
```

This way, if you're reading the doc on Github you get redirected within Github. If you're reading on Gitbook, you get redirected within Gitbook

**Comment by bstuder99** [comment]
Thomas, I'm fine with that change generally but when I try to do that here, it 404s with https://github.com/kubecost/docs/blob/fed-etl-updates/federated-etl-backups-alerting.md. I'm hoping when we merge the branch (where it thinks this doc will be located), that problem will resolve itself. If not, I can correct the error post-merge.

**Review by bstuder99 — APPROVED** [review]

**Comment by thomasvn** [comment]
@bstuder99 Great! I'm good with that!

**Pull request merged**